packages/story-editor/src/utils/usePageAsCanvas.js

```diff
@@ -95,9 +98,15 @@ function usePageAsCanvas() {
         if (!pageCanvasPromise && fullbleedContainer) {
           import(/* webpackChunkName: "chunk-html-to-image" */ 'html-to-image')
             .then((htmlToImage) => {
+              const filter = (node) => {
+                // exclude selected element from generated image to prevent interfereing with contrast calculation. true includes and false excludes
+                return node?.dataset?.elementId !== selectedElementIds[0];
```

Comment:
@Trapsta We still need to add a flag for skipping the element, it shouldn't be always skipped, only if we're using the "Magic wand" button. So let's add `skipSelectedElement` or something like that as a boolean to the `usePageAsCanvas` hook, and only filter if that is set.